cabot_alert_hipchat/models.py

```diff
@@ -45,21 +96,8 @@ def send_alert(self, service, users, duty_officers):
         })
         message = Template(hipchat_template).render(c)
         self._send_hipchat_alert(message, color=color, sender='Cabot/%s' % service.name)
+        self._send_images_to_hipchat(service)
```

Comment:
Oh. In such a case, let us send the message first and then follow it up with images. Also helps us to ensure that we handle image fetches gracefully. Let us not club everything.
Text + n * img